app/src/main/java/edu/uc/reedws/musiclink/ui/main/ApplicationViewModel.kt

```diff
@@ -17,16 +17,16 @@ class ApplicationViewModel(application: Application): AndroidViewModel(applicati
         fetchPlaylists()
     }
 
-    fun fetchPlaylists() {
-        viewModelScope.launch {
-            playlists = _playlistService.fetchPlaylists()
-        }
-    }
-
     fun createPlaylist(name: String) {
         viewModelScope.launch(Dispatchers.IO) {
             _playlistService.createPlaylist(name)
             fetchPlaylists()
         }
     }
+
+    private fun fetchPlaylists() {
+        viewModelScope.launch {
+            playlists = _playlistService.fetchPlaylists()
+        }
+    }
```

Comment:
I changed the visibility modifier of fetchPlaylists() to private because this function is only being used inside the ApplicationMainViewModel class and nowhere else.